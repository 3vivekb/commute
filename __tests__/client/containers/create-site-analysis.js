/* global describe, expect, it */

import {mount} from 'enzyme'
import {mountToJson} from 'enzyme-to-json'
import React from 'react'
import {Provider} from 'react-redux'

import {makeMockStore, mockStoreData} from '../../test-utils/mock-store.js'

import CreateSiteAnalysis from '../../../client/containers/create-site-analysis'

const mockStore = makeMockStore(mockStoreData)

describe('Container > CreateSiteAnalysis', () => {
  it('renders correctly', () => {
    // mount component
    const tree = mount(
      <Provider store={mockStore}>
        <CreateSiteAnalysis />
      </Provider>
    )
    expect(mountToJson(tree.find(CreateSiteAnalysis))).toMatchSnapshot()
  })
})
