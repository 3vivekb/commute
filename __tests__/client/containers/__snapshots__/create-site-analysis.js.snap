exports[`Container > CreateSiteAnalysis renders correctly 1`] = `
<Connect(CreateSiteAnalysis)>
  <CreateSiteAnalysis
    dispatch={[Function]}>
    <h1>
      Create Site Analysis
    </h1>
  </CreateSiteAnalysis>
</Connect(CreateSiteAnalysis)>
`;
